Tests/TestMessagesViewControllerModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TestMessagesViewControllerModel.swift
+//  MessageKit
+//
+//  Created by Dan Leonard on 8/14/17.
+//  Copyright © 2017 MessageKit. All rights reserved.
+//
```

Comment:
Check this post https://robots.thoughtbot.com/creating-custom-xcode-templates, it's possible to create a custom template add MIT license.